### ISSUE TITLE
News Archive links to https://labstalk.buzzsprout.com give 404 error

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#2112]
## Where to find the issue

- https://www.coronawarn.app/en/blog/archive/ "News Archive"
- https://www.coronawarn.app/de/blog/archiv/ "News Archiv"

## Describe the issue

Labs Talk links are now producing a 404 file not found error.

- https://labstalk.buzzsprout.com/993481/4500305-corona-warn-app-update-with-malte-janduda "Labs Talk - Corona-Warn-App Update with Malte Janduda" @Maltej
- https://labstalk.buzzsprout.com/993481/4105496-the-corona-warn-app-with-thomas-klingbeil-sap "Labs Talk: The Corona-Warn-App with Thomas Klingbeil, SAP" @tklingbeil

From `cwa-website/cypress/logs/broken_links_result.txt`:

```
==================== Broken links ====================

[RESPONSE 404] https://labstalk.buzzsprout.com/993481/4500305-corona-warn-app-update-with-malte-janduda on '/de/blog/archiv/' 
[RESPONSE 404] https://labstalk.buzzsprout.com/993481/4105496-the-corona-warn-app-with-thomas-klingbeil-sap on '/de/blog/archiv/' 
[RESPONSE 404] https://labstalk.buzzsprout.com/993481/4500305-corona-warn-app-update-with-malte-janduda on '/en/blog/archive/' 
[RESPONSE 404] https://labstalk.buzzsprout.com/993481/4105496-the-corona-warn-app-with-thomas-klingbeil-sap on '/en/blog/archive/' 
```

![image](https://user-images.githubusercontent.com/66998419/142990441-d8dd0943-108f-4ccf-a9df-7db794f9cab5.png)

## Suggested change

1. Investigate if this is a temporary issue on the https://labstalk.buzzsprout.com site or a permanent change.
2. If it is a permanent change, then either rehost the podcasts or remove them from https://www.coronawarn.app.

## Additional information

This is a success story for Cypress [check_links.js](https://github.com/corona-warn-app/cwa-website/blob/master/cypress/integration/check_links.js)! I only noticed this issue by running the Cypress test suite on cwa-website.

**Comment by MikeMcC399** [comment]
It seems that the domain https://labstalk.buzzsprout.com/ was an SAP-specific presence on the platform https://buzzsprout.com/ and that this is no longer functional.

Perhaps SAP has cancelled their presence and subscription to this service?

In any case, so that there are no dead links on the current production web https://www.coronawarn.app I submitted PR #2115 to remove the non-working blog links.

**Comment by MikeMcC399** [comment]
This issue is resolved by PR #2115.

If a new location is found for the deleted podcasts, or of the old location is reactivated, they can be added again.
